rcssserver3d/data/rsg/agent/ball.rsg

```diff
@@ -2,12 +2,13 @@
 
 (RSG 0 1)
 (
+ (templ $X $Y)
  (def $Radius (eval Soccer.BallRadius))
  (def $Material matOrange)
  (def $Mass (eval Soccer.BallMass))
 
  (nd Ball
-     (setLocalPos 0.0 0.0 $Radius)
+     (setLocalPos $X $Y $Radius)
```

Comment:
is it possible to define defaults for X and Y?  like Radius?

Reply:
i think you can do something like `(def $X (eval $X || 0))`

Reply:
Mmm. perhaps not? Getting
`parameter value 'X' not supplied`

Reply:
can we make a separate rsg file then?  like `beamed-ball.rsg`?

Reply:
Either I don't understand what your suggesting or I don't know what we gain by doing this?

Reply:
if and when we ever resync with upstream, if they have made changes to ball.rsg, then we haven't interfered with them.  or if they have created new games using ball.rsg, then those new games will work after a merge.

Reply:
We've already made changes to this file anyway (ball physics tweaks)
I don't think it's a good idea to avoid merge conflicts like this. We'll end up with duplicate files which are more difficult to maintain. IMO when merging, this specific change should be ported across to all "new games"